.eslintrc.json

```diff
@@ -85,7 +85,7 @@
     "comma-dangle": ["error", "always-multiline"],
     "import/no-extraneous-dependencies": [
       "error",
-      { "devDependencies": ["**/*.test.js", "**/*.test.ts", "**/testutils/*.js"] }
+      { "devDependencies": ["**/*.test.js", "**/*.test.ts", "**/testutils/*.js", "cypress.config.ts"] }
```

Comment:
was picking up the cypress config here causing the linting to fail as it then preferred cypress over jest